Docs/reference/content/upgrading.md

```diff
@@ -11,9 +11,6 @@ title = "Upgrading"
 
 ## Breaking Changes
 
-For most users there should be no breaking changes in version 2.8.0 of the driver.
+There should be no breaking changes in version 2.9.0 of the driver.
```

Comment:
As I see, the default values change is not a breaking change (Since it will not be a reason for app fail). 
If some other tickets change it, this line should be updated.